README.md

```diff
@@ -1,6 +1,10 @@
 # LLaMPPL + HuggingFace
 
 [![docs](https://github.com/probcomp/hfppl/actions/workflows/docs.yml/badge.svg)](https://probcomp.github.io/hfppl)
+[![Tests](https://github.com/probcomp/hfppl/actions/workflows/pytest.yml/badge.svg)](https://github.com/probcomp/hfppl/actions/workflows/pytest.yml)
+[![codecov](https://codecov.io/gh/probcomp/hfppl/graph/badge.svg?token=414EHUC2P3)](https://codecov.io/gh/probcomp/hfppl)
```

Comment:
Just confirming that `token` isn't intended to be secure?

Reply:
yep that's correct, this is what codecov sends us to copy-paste.